### ISSUE TITLE
Country Map visualisation for UK - zoomed out

### ISSUE THREAD

**Issue by liucijalat** [apache/superset#4568]
Make sure these boxes are checked before submitting your issue - thank you!

- [X] I have checked the superset logs for python stacktraces and included it here as text if any
- [X] I have reproduced the issue with at least the latest released version of superset
- [X] I have checked the issue tracker for the same issue and I haven't found one similar

### Superset version
0.23.2

### Expected results
A blank zoomed in map of UK

### Actual results
A zoomed out map of UK with a square frame around it which shows 'Isle of Man' when hovering over it
<img width="1662" alt="screen shot 2018-03-07 at 20 33 15" src="https://user-images.githubusercontent.com/32986613/37117107-3010d20c-2248-11e8-84d9-bc4aa2265859.png">

### Steps to reproduce
1. Open the Misc Charts dashboard from the sample dashboards provided when installing Superset
2. Click 'Explore chart' on the 'Birth in France by department in 2016' widget
3. Under Options, change the Country Name to 'Uk' and run query

**Comment by liucijalat** [comment]
I found that removing Isle of Man from the uk.geojson file solves the problem. The https://github.com/apache/incubator-superset/pull/3911 pull request added Isle of Man. @alanmcruickshank did you have this problem?

**Comment by mistercrunch** [comment]
@alanmcruickshank

**Comment by dominijk** [comment]
@mistercrunch how does the code pick the geosjon file?  if I have more than one geosjosn file with the same ISO codes (ie Uk and then London with more detailed polygon) will it pick one over the other or is it just scanning for codes. In the chart json object I can see the below line where I could hard code a path, but am i right in thinking it will overwrite if I make an edit?

```JSON
  "select_country": "Uk",
  "number_format": ".3s",
```


@liucijalat did you simply just delete out the Isle of Man object and put that back on your server / was your install docker or on a server?

**Comment by dominijk** [comment]
@mistercrunch We've successfully added a city as a geojson as london.geosjson and then got this to display by adjusting the parameters in the chart metadata, when we then tested adding the country list as [docs](https://superset.incubator.apache.org/visualization.html#need-to-add-a-new-country) it didn't appear. Hacking the Uk file to just contain London geom did however work. Is there a constraint that stops us adding countries that aren't countries?
